assets/js/components/PageHeader/PageHeader.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function PageHeader({ children }) {
```

Comment:
You can pass `className` as a prop and combine classes using the `classnames` library.

By doing this you can avoid using `<span className="font-bold"></span>` and you can instead write `<PageHeader className="font-bold"></PageHeader>`

Reply:
I can pass the className prop to the component, that I think the `<span className="font-bold"></span>` is needed anways as for some views, e.g. `ClusterDetails` it's needed to have one part without bold while the cluster name goes in bold. Wdyt, is it still worth to pass it?

Reply:
Yes anyway it's worth to pass it, `className` is sort of _mandatory_ for every component one makes because it dramatically increases composability, especially for this pure UI stuff

Reply:
should be good now, thanks!

Reply:
Excellent! Now this

```js
<PageHeader>
  <span className="font-bold">Checks catalog</span>
</PageHeader>
```

can be just

```js
<PageHeader>Checks catalog</PageHeader>
```

Reply:
Yes, I missed that, thanks!